docs/notifications.md

```diff
@@ -0,0 +1,122 @@
+# Notifications
+
+Key to workflows is receiving timely notifications in the places where you work. This module provides a developer framework for creating notifications, either as hardcoded business logic or by making events, messages, recipients and destinations available to the Workflow builder UI.
+
+This component is powered by the [Workflows plugin](https://github.com/humanmade/Workflows). You can find more [detailed documentation on the plugin's wiki](https://github.com/humanmade/Workflows/wiki).
+
+## Basic hardcoded example
```

Comment:
So can could this notification also be created via the UI? The later docs allude to be able to use the UI to construct this kind of thing. It might be good to make that explicit here?